wifiphisher/common/constants.py

```diff
@@ -85,6 +85,38 @@
 GR = '\033[37m'  # gray
 T = '\033[93m'   # tan
 
+# Logging configurations
+# possible values for debug levels are:
+# CRITICAL, ERROR, WARNING, INFO, DEBUG, NOTSET
+LOG_LEVEL = 'INFO'
+LOG_FILEPATH = 'wifiphisher.log'
+LOGGING_CONFIG = {
+    'version': 1,
+    # Defined the handlers
+    'handlers': {
+        'file': {
+            'class': 'logging.handlers.RotatingFileHandler',
+            'level': LOG_LEVEL,
+            'formatter': 'detailed',
+            'filename': LOG_FILEPATH,
+            'backupCount': 3,
+        },
+    },
+    # fomatters for the handlers
+    'formatters': {
+        'detailed': {
+            'format': '%(asctime)s - %(name)-32s - %(levelname)-8s - %(message)s'
```

Comment:
@anakin1028 Is there a specific reason you have added so many spaces between the name and the level?

Reply:
I just try to make the logging field more neatly since the length of module name and the length of the level may be different.  do you think the original format is better?

Reply:
I do think that 32 spaces is a bit of a exaggeration. Maybe 4 would be better.

Reply:
`-32` means the `spaces + module-name`. So for example if we have `wifiphisher.extensions.deauth` then it will add 3 spaces before the module name